### PR TITLE
Weather App

### PR DESCRIPTION
**Pull request by JHarde**
Link to my deployed project: https://stockholmweather.netlify.app/

My reflections on how this week's project turned out: This wasn't my best week, struggled a lot due to lack of time because of work and had a hard time figuring some things out. I would like to make my app better later on and strive towards red and black level. 

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: Code/script.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    description.innerHTML = json.weather.map((a) => a.description);
+    
+
+    sun.innerHTML += `<p id="sunrise">Sunrise ${`${sunrise.getHours()}:${sunrise.getMinutes()}`}</p>`;
+    sun.innerHTML += `<p id ="sunset">Sunset ${`${sunset.getHours()}:${sunset.getMinutes()}`}</p>`;
+
+    const newDate = new Date(json.dt * 1000)
+    const todaysDate = newDate.toDateString()
+    document.getElementById("date").innerHTML = `${todaysDate}`;
+
```
Comment (LillanEriksson) [inline Code/script.js:51]: Very neat and readable code Joel!

File: Code/script.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    }
+
+ const getNumber = (theNumber) => {
+    if (theNumber > 0) {
+      return theNumber;
+    } else {
+      return theNumber.toString();
+    }
+  }
+
```
Comment (LillanEriksson) [inline Code/script.js:104]: I have really nothing to comment about, except that you could have put more into different functions.

File: Code/style.css [hunk 3]
```diff
@@ -0,0 +1,187 @@
+
+
+/* Extra small devices (phones, 600px and down) */
+@media only screen and (max-width: 600px) {
+
```
Comment (LillanEriksson) [inline Code/style.css:5]: The responsivity looks great on the live page, you did a great job with that!

File: Code/style.css [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+    
+#weekdays {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    text-align: center;
+    color: #2A5510;
+    font-size: 45px;
+}
```
Comment (LillanEriksson) [inline Code/style.css:186]: I think it's preffered to style by class and stick with that, and not mix styling ids and classes. I think the coaches have mentioned this during some lectures too.

**Review by LillanEriksson — APPROVED** [review]
The page looks good Joel, and I totally understand that you had a timeshort this week. and I think you have reached the blue level.

**Review by EmOlsz — APPROVED** [review]
Nice project Joel, congratulations! It's working (so that's most important) and looking good. Nonetheless, I see that your code starts to be messy - it seems that you haven't checked it before final push.
Almost all of my comments are about possible improvements, please consider them as very important, because time to change some tendencies of yours is right now (and will end soon as they will change from tendencies to habits). Provide those changes and then I'll approve your project.
Apart from that, as I said, good job :)

**Review by EmOlsz — CHANGES_REQUESTED** [review]
Changes requested

**Comment by JHarde** [comment]
Hi Maks!

I have looked through the changes you requested and tried my best to fix them. Regarding the indentation I have tried to use "Prettier" but for some reason it doesn't seem to work for me in this project. Worked well last week. Feel free to let me know more in detail how I can change this if changes still need to be made.

**Comment by EmOlsz** [comment]
> Hi Maks!
> 
> I have looked through the changes you requested and tried my best to fix them. Regarding the indentation I have tried to use "Prettier" but for some reason it doesn't seem to work for me in this project. Worked well last week. Feel free to let me know more in detail how I can change this if changes still need to be made.

Hey Joel! Thank you for quick changes, everything is ok now :)
Regarding Prettier - well, shit happens unfortunately. This software is not intuitive at all and breaks a lot. You don't have to worry about that in this case!

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**